commit 5a9973c383f4b94d56b3b51ceb90166c97615809
Author: Michael Still <mikal@stillhq.com>
Date:   Fri Nov 21 12:46:44 2025 +1100

    Make fluentd optional.
    
    The refactoring of fluentd out of the common role into its own
    role appears to have missed supporting enable_fluentd being set
    to "no".
    
    Change-Id: I4b71af89775569e871d18ef60e5200f22c9699a9
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/site.yml b/ansible/site.yml
index ae5f281cb..ad963e513 100644
--- a/ansible/site.yml
+++ b/ansible/site.yml
@@ -121,10 +121,11 @@
     {{ fluentd_max_fail_percentage |
        default(kolla_max_fail_percentage) |
        default(100) }}
-  tags:
-    - fluentd
-  roles:
-    - role: fluentd
+  tasks:
+    - include_role:
+        name: fluentd
+      tags: fluentd
+      when: enable_fluentd | bool
 
 - name: Apply role loadbalancer
   gather_facts: false
